test/uk/gov/hmrc/helptosavereminder/controllers/EmailCallbackControllerSpec.scala

```diff
@@ -148,28 +149,33 @@ class EmailCallbackControllerSpec extends UnitSpec with MongoSpecSupport with Gu
     }
   }
 
-  "respond with a 200 containing FAILURE string if Nino does not exists or update fails" in {
+  /*"respond with a 200 containing FAILURE string if Nino does not exists or update fails" in {
```

Comment:
Why is this test commented?

Reply:
Changed to a Future that is successful but throwns the required error message to the caller. And uncommented the test case. Done.